src/routes/projection/page.css

```diff
@@ -0,0 +1,3 @@
+html {
+  background-color: #262626;
+}
```

Comment:
This might warrant deciding if we want to have CSS declarations done "inline" via a `<style>` tag or not. Mostly a preference thing and I don't necessarily care either way. (It's also possible this declaration had to be put externally due to the scoping)

Reply:
That all makes sense to me, I wasn't certain what preference if any there was when applying these sorts of things.